ground/gcs/src/plugins/config/configautotunewidget.cpp

```diff
@@ -11,16 +38,21 @@
 #include <QDesktopServices>
 #include <QUrl>
 #include <QList>
+#include <QMessageBox>
+#include <QClipboard>
+#include <QtNetwork/QNetworkAccessManager>
+#include <QtNetwork/QNetworkRequest>
 #include "systemident.h"
 #include "stabilizationsettings.h"
 #include "modulesettings.h"
+#include "systemsettings.h"
 #include "coreplugin/generalsettings.h"
-#include "utils/phpbb.h"
 #include "extensionsystem/pluginmanager.h"
-#include <QMessageBox>
+#include "coreplugin/iboardtype.h"
+
+
+const QString ConfigAutotuneWidget::databaseUrl = QString("http://dronin-autotown.appspot.com/storeTune");
```

Comment:
Yeah, I was thinking in addition to moving the URL I'd like to wrap some JSON request functionality into a util class. ;-)